File: Exercise/Hard/13. Excel Column Number/Code/Excel Column Number.cs
using System;

public class Program {
    public static int TitleToNumber(string s) {
        int result = 0;
        foreach (char c in s) {
            result = result * 26 + (c - 'A' + 1);
        }
        return result;
    }

    public static void Main() {
        string s = Console.ReadLine();
        
        int number = TitleToNumber(s);
        Console.WriteLine(number);
    }
}

File: Exercise/Hard/13. Excel Column Number/Code/Excel Column Number.js
function titleToNumber(s) {
    let result = 0;
    for (let i = 0; i < s.length; i++) {
        result = result * 26 + (s.charCodeAt(i) - 'A'.charCodeAt(0) + 1);
    }
    return result;
}

console.log(titleToNumber("A"));   
console.log(titleToNumber("AB"));  

